hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -80,51 +84,63 @@
         { bits: "15:8",
           name: "HW_REVISION",
           desc: "Read of this register shows the revision of this block.",
-          resval: "0x1"
-       }
+          resval: "0x3"
+        }
         { bits: "7:0",
           name: "ABI_REVISION",
           desc: "Read of this register shows the ABI of this block.",
-          resval: "0x1"
+          resval: "0x3"
         }
       ]
     },
-    { name: "ES_CONF",
-      desc: "ENTROPY_SRC configuration register",
+    { name: "CONF",
+      desc: "Configuration register",
       swaccess: "rw",
       hwaccess: "hro",
-      regwen:   "ES_REGEN",
+      regwen:   "REGEN",
       fields: [
         { bits: "1:0",
           name: "ENABLE",
-          desc: '''Primary enable for the ENTROPY_SRC entropy generation function.
+          desc: '''Module enable for the ENTROPY_SRC entropy generation function.
                 This two bit field determines what source will be used for health
                 check processing:
                 0b00: Disabled
-                0b01: LFSR mode enabled
-                0b10: RNG mode enabled
+                0b01: RNG mode enabled
```

Comment:
Two things occur to me here, and since this the first file I am re-reviewing, putting a comment down to remind me (I may find the answer later along):

1) It might be good if this 2-bit signal was output from the entropy complex (not sure where to yet) as this choice may be a good thing to attest to in terms of device configuration/system state.
2) Does this signal truly only affect the health check (as described on line 105/106) or is it affecting all of the entropy generation pipeline?